qt/settingsdialog.py

```diff
@@ -2373,9 +2373,14 @@ def searchConfig(self, path):
 
                 try:
                     cfg = config.Config(cfgPath)
+
                     if cfg.isConfigured():
                         return cfg
-                except:
+
+                except Exception as exc:
```

Comment:
Catching all exception but doing nothing about it (`pass`). Can not imagine how much bugs this may hide or trigger.